targettest/sym1/symIO.c

```diff
@@ -0,0 +1,168 @@
+         puts( "IOR2A, DDR2B, IOR2B, DDR3A, IOR3A, DDR3B and IOR3B." );
+         instr = 0;
+      }
+
+      printf( "\n Command:  " );
+
+      fgets(cmd, sizeof(cmd)-1, stdin);
+      cmd[strlen(cmd)-1] = '\0';
+
+      if( strncasecmp(cmd, "quit", 4) == 0) {
+         going = 0;  
+      }
+      else if( strncasecmp(cmd, "help", 4) == 0) {
+         instr = 1;
+      }
+      else if( strncasecmp(cmd, "ddr1a", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = (unsigned char) strtol( vp, NULL, 0 );
+            VIA1.ddra = val;
+         }
+      }
+      else if( strncasecmp(cmd, "ior1a", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = (unsigned char) strtol( vp, NULL, 0 );
+            VIA1.pra = val;
+         }
+      }
+      else if( strncasecmp(cmd, "ddr1b", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = (unsigned char) strtol( vp, NULL, 0 );
+            VIA1.ddrb = val;
+         }
+      }
+      else if( strncasecmp(cmd, "ior1b", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = (unsigned char) strtol( vp, NULL, 0 );
+            VIA1.prb = val;
+         }
+      }
+      else if( strncasecmp(cmd, "ddr2a", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = (unsigned char) strtol( vp, NULL, 0 );
+            VIA2.ddra = val;
+         }
+      }
+      else if( strncasecmp(cmd, "ior2a", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = (unsigned char) strtol( vp, NULL, 0 );
+            VIA2.pra = val;
+         }
+      }
+      else if( strncasecmp(cmd, "ddr2b", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = (unsigned char) strtol( vp, NULL, 0 );
+            VIA2.ddrb = val;
+         }
+      }
+      else if( strncasecmp(cmd, "ior2b", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = (unsigned char) strtol( vp, NULL, 0 );
+            VIA2.prb = val;
+         }
+      }
+      else if( strncasecmp(cmd, "ddr3a", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = (unsigned char) strtol( vp, NULL, 0 );
+            VIA3.ddra = val;
+         }
+      }
+      else if( strncasecmp(cmd, "ior3a", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = (unsigned char) strtol( vp, NULL, 0 );
+            VIA3.pra = val;
+         }
+      }
+      else if( strncasecmp(cmd, "ddr3b", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = (unsigned char) strtol( vp, NULL, 0 );
+            VIA3.ddrb = val;
+         }
+      }
+      else if( strncasecmp(cmd, "ior3b", 5) == 0) {
+         vp = strchr(cmd, ' ');
+         if( vp ) {
+            val = (unsigned char) strtol( vp, NULL, 0 );
+            VIA3.prb = val;
+         }
+      }
+   }
+}
```

Comment:
Please adhere to the cc65 code style, see symDisplay.c.

Reply:
Updated.  Please check and confirm.

Reply:
Please adhere to the cc65 code style, see symDisplay.c.